bin/gftools-add-designer.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python3
+"""Add or update a designer entry in the Google Fonts catalog.
+
+Designer profiles are stored in the google/fonts repo:
+https://github.com/google/fonts/tree/main/catalog/designers
+
+This script is intended to help onboarders quickly create individual designer
+profiles.
+
+Designer Profiles are visible to the public. They are included in the the about
+section for each family e.g:
+https://fonts.google.com/specimen/Roboto#about
+
+In 2021, Rosalie Wagner created an online form for us to collect designer
+information from the general public. The form populates the following spreadsheet,
+https://docs.google.com/spreadsheets/d/1G1rkk_jJnuV7lVWkfCbqwTkSjbbWJcC1a2Xfoh-nRWM/edit#gid=1905114301
+
+There is also an image directory where the designer's profile pics are uploaded into.
+
+In order to use the spreadsheet command in this script, you will need to
+download the folder which contains the spreadsheet and designer images.
+You can then use the ``--spreadsheet`` arg.
+
+Usage:
+# Add or update a designer entry. User will need to hand complete the bio.html
+$ gftools add-designer ~/Type/fonts/catalog/designers "Theo Salvadore" path/to/img.png
+
+# Add or update a designer entry and use the spreadsheet to create the bio.html file
```

Comment:
Change "Add or update a designer entry and use the spreadsheet to create the bio.html file" to "Add or update a designer entry using the spreadsheet."